src/sortStudents.ts

```diff
@@ -1,16 +1,53 @@
-
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
+}
+
+export type SortOrder = 'asc' | 'desc';
+
+function getAverageGrade({ grades }: Student): number {
```

Comment:
Why pass an entire student?
```suggestion
function getAverageGrade(grades: number[]): number {
```